src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import core.basesyntax.interfaces.AreaCalculator;
+import core.basesyntax.interfaces.Describer;
+
+public abstract class Figure implements Describer, AreaCalculator {
```

Comment:
Avoid using abstract classes to set behavior for classes. Instead, consider using interfaces if you need to define a contract for classes or abstract classes if there are some common fields or methods that can be shared across all subclasses.